packages/@n8n/nodes-langchain/nodes/vector_store/VectorStorePGVector/VectorStorePGVector.node.ts

```diff
@@ -307,6 +308,11 @@ export class VectorStorePGVector extends createVectorStoreNode({
 			metadataColumnName: 'metadata',
 		}) as ColumnOptions;
 
-		await PGVectorStore.fromDocuments(documents, embeddings, config);
+		const vectorStore = await PGVectorStore.fromDocuments(documents, embeddings, config);
+		vectorStore.client?.release();
+	},
+
+	releaseVectorStoreClient(vectorStore) {
+		(vectorStore as PGVectorStore).client?.release();
```

Comment:
```suggestion
	releaseVectorStoreClient(vectorStore) {
		if (vectorStore instanceof PGVectorStore) {
			vectorStore.client?.release();
```

Would that also work?